components/tools/OmeroWeb/omeroweb/feedback/views.py

```diff
@@ -180,7 +167,11 @@ def handler500(request):
     if request.is_ajax():
         return HttpResponseServerError(error500)
 
-    return custom_server_error(request, error500)
+    form = ErrorForm(initial={'error': error500})
```

Comment:
I doubt anyone is using `custom_server_error()` but is there a reason why this was removed and the code copied into this handler?